postmanparser/url.py

```diff
@@ -25,9 +25,15 @@ def parse(cls, data: dict):
             port = "80"
         if protocol == "https":
             port = "443"
+
         query = data.get("query", None)
         if query:
             query = [KeyVal.parse(_) for _ in query]
+
+        variable = data.get("variable", None)
```

Comment:
Do `variable = data.get("variable", [])` instead of none, default should be empty list.